app/src/test/kotlin/compiler/intermediate/generators/DefaultFunctionDetailsGeneratorTest.kt

```diff
@@ -20,6 +20,9 @@ class DefaultFunctionDetailsGeneratorTest {
     private val resultDummyVariable = Variable(Variable.Kind.VALUE, "numberResult", Type.Number, null)
     private val resultVariableRegister = Register()
 
+    private val callerSavedRegisters = listOf(Register.RAX, Register.RCX, Register.RDX, Register.RSI, Register.RDI, Register.R8, Register.R9, Register.R10, Register.R11)
```

Comment:
Why copy the list from `SysV64CallingConvention`?

Reply:
The list in `SysV64CallingConvention` is part of the very implementation we want to test. Let's imagine that someone mistakenly changed the list in SysV64CallingConvention.kt. In that case, some unit tests should fail but if we imported the list in test files, we wouldn't detect that.